lib/pages/connect/connect_page_mixin.dart

```diff
@@ -143,4 +143,15 @@ mixin ConnectPageMixin {
     }
     return list;
   }
+
+  bool isSingleAccountOnHomeserver(
+    MatrixState matrix,
+    Uri homeserver,
+  ) {
+    if (matrix.client.isLogged() && matrix.client.homeserver == homeserver) {
+      return true;
+    } else {
+      return false;
+    }
```

Comment:
return matrix.client.isLogged() && matrix.client.homeserver == homeserver;

Reply:
i think you should make a extension for client `isSingleAccountOnHomeserver`, instead of put it in mixin